integration/mediation-tests/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/clients/stockquoteclient/StockQuoteClient.java

```diff
@@ -280,7 +280,7 @@ private ServiceClient getServiceClient(String trpUrl, String addUrl, String oper
         if (httpHeaders.size() > 0) {
             options.setProperty(HTTPConstants.HTTP_HEADERS, httpHeaders);
         }
-        options.setTimeOutInMilliSeconds(45000);
+        options.setTimeOutInMilliSeconds(60000);
```

Comment:
let's define constant for the time period

Reply:
Fixed. Please refer latest commit.